src/components/blog-page/blog-tags.tsx

```diff
@@ -25,7 +25,12 @@ export function BlogTags({
         }
 
         const tagElement = (
-          <Tag className={opaque ? "bg-neu-0" : ""} color={color}>
+          // conditionally add a key here to avoid react list key errors
+          <Tag
+            key={!links ? tag : undefined}
+            className={opaque ? "bg-neu-0" : ""}
+            color={color}
+          >
```

Comment:
oh I can't believe I missed that

mby we could shorten it and add the tag unconditionally

it's the only child so even if it was a long string React won't spend a lot of time comparing it

```suggestion
          <Tag
            key={tag}
            className={opaque ? "bg-neu-0" : ""}
            color={color}
          >
```